[3.8] Fix typos in Misc/NEWS.d

toonarmycaptain opened the pull request:
With reference to corrections to [news in 3.9.0a2.rst](https://github.com/python/cpython/pull/17665) backporting some typo corrections to 3.8.0/3.8.1.

toonarmycaptain committed:
Backport typo corrections.

toonarmycaptain committed:
Update 3.8.0.rst

aeros reviewed (changes requested):
Thanks for the PR @toonarmycaptain.

Could you prefix the PR title with `[3.8]`? This would look like: `[3.8] Backport typo corrections`. This format is required for any PR opened against a maintenance branch, otherwise it won't pass @bedevere-bot's checks. 

I would also consider adjusting to title to something more informative of the PR, such as `[3.8] Fix typos in Misc/NEWS.d`. "typo corrections" doesn't explain much about the location of the changes, as that could apply to any part of the documentation, code comments, docstrings, etc. 

I believe it is correct to specifically open the PR against the 3.8 branch, as changes to Misc/NEWS.d are typically only applied to branch's versions rather than all versions. This differs from other areas, where the changes should be applied to master and then backported to previous versions to all relevant locations.

I'm not certain about the typo fix that adjusts the number of spaces between sentences. IIRC, it still renders exactly the same in reST; @merwok or @willingc might be able to clarify on that. 

The other changes LGTM though, thanks!

toonarmycaptain committed:
Update 3.8.0.rst

Reverse deletion of extra space between sentences added solely for consistency.

toonarmycaptain committed:
Update 3.8.0.rst

Reverse deletion of extra space between sentences added solely for consistency.

...apparently there were three spaces originally?
